### PR TITLE
Uploading news's image.

### PR DESCRIPTION
**Pull request by MateuszSz93**

### CODE REVIEW

File: src/main/java/com/intive/patronage/toz/news/NewsService.java [hunk 1]
```diff
@@ -84,4 +84,11 @@ News updateNews(final UUID id, final News news) {
         }
         return newsList;
     }
+
+    void updateNewsImageUrl(final UUID id, String photoUrl) {
+        RepositoryChecker.throwNotFoundExceptionIfNotExists(id, newsRepository, NEWS);
```
Comment (mlotysz) [inline src/main/java/com/intive/patronage/toz/news/NewsService.java]: So you doing one find in DB to check existence of NEWS with `id` and in following line you doing findOne. You could be better than this and I'm sure you find a way to save one db operation here? :)

File: src/main/java/com/intive/patronage/toz/news/NewsService.java [hunk 2]
```diff
@@ -84,4 +85,13 @@ News updateNews(final UUID id, final News news) {
         }
         return newsList;
     }
+
+    void updateNewsImageUrl(final UUID id, String photoUrl) {
+        final News news = newsRepository.findOne(id);
+        if (news == null){
```
Comment (maras93) [inline src/main/java/com/intive/patronage/toz/news/NewsService.java]: Use auto format code.

File: src/main/java/com/intive/patronage/toz/news/NewsController.java [hunk 3]
```diff
@@ -118,4 +129,26 @@ public NewsView getNewsById(@PathVariable UUID id) {
...
+    @ApiOperation(value = "Upload image", notes =
+            "Required roles: SA, TOZ.")
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "Bad Request"),
+            @ApiResponse(code = 422, message = "Invalid image file")
+    })
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping(value = "/{id}/images", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    @PreAuthorize("hasAnyAuthority('SA', 'TOZ')")
+    public ResponseEntity<UrlView> uploadFile(@PathVariable UUID id, @RequestParam("file") MultipartFile file) {
```
Comment (maras93) [inline src/main/java/com/intive/patronage/toz/news/NewsController.java]: I think Param `"file"` from `@RequestParam` could be removed.

File: src/main/java/com/intive/patronage/toz/pet/PetsController.java [hunk 4]
```diff
@@ -140,27 +141,15 @@ public PetView updatePet(@PathVariable UUID id, @Valid @RequestBody PetRequestBo
     @PostMapping(value = "/{id}/images", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
     @PreAuthorize("hasAnyAuthority('SA', 'TOZ')")
     public ResponseEntity<UrlView> uploadFile(@PathVariable UUID id, @RequestParam("file") MultipartFile file) {
```
Comment (maras93) [inline src/main/java/com/intive/patronage/toz/pet/PetsController.java]: As above

File: src/main/java/com/intive/patronage/toz/pet/PetsService.java [hunk 5]
```diff
@@ -48,8 +49,10 @@ Pet updatePet(final UUID id, final Pet pet) {
     }
 
     void updatePetImageUrl(final UUID id, String imageUrl) {
-        RepositoryChecker.throwNotFoundExceptionIfNotExists(id, petsRepository, PET);
         final Pet pet = petsRepository.findOne(id);
+        if (pet == null){
```
Comment (maras93) [inline src/main/java/com/intive/patronage/toz/pet/PetsService.java]: Reformat code.

File: src/main/java/com/intive/patronage/toz/news/NewsController.java [hunk 6]
```diff
@@ -139,7 +139,7 @@ private static News convertFromView(final NewsView newsView) {
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping(value = "/{id}/images", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
     @PreAuthorize("hasAnyAuthority('SA', 'TOZ')")
-    public ResponseEntity<UrlView> uploadFile(@PathVariable UUID id, @RequestParam("file") MultipartFile file) {
+    public ResponseEntity<UrlView> uploadFile(@PathVariable UUID id, @RequestParam() MultipartFile file) {
```
Comment (maras93) [inline src/main/java/com/intive/patronage/toz/news/NewsController.java]: Brackets also could be removed 😄

Comment (MateuszSz93) [inline src/main/java/com/intive/patronage/toz/news/NewsController.java]: Little oversight :P

**Pull request merged**